src/pseudopeople/configuration/generator.py

```diff
@@ -75,12 +75,15 @@ def get_configuration(user_configuration: Union[Path, str, Dict] = None) -> Conf
     :param user_configuration: A path to the YAML file or a dictionary defining user overrides for the defaults
     :return: a ConfigTree object of the noising configuration
     """
-
-    if type(user_configuration) == str and user_configuration.lower() == Keys.NO_NOISE:
-        config_type = Keys.NO_NOISE
+    if isinstance(user_configuration, str) and user_configuration.lower() == NO_NOISE:
+        config_type = NO_NOISE
         user_configuration = None
+    elif isinstance(user_configuration, (Path, str)):
```

Comment:
I'm not sure it's necessary immediately, but the DEFAULT and NO_NOISE strings are confusing. I'm afraid that a user will pass in `user_configuration = DEFAULT` which would break here b/c it would try and load a yaml file at the path "default".

Reply:
I do agree its a bit confusing but not sure what placeholder value we should use if it is not NO_NOISE